src/cmd/cli/command/commands_test.go

```diff
@@ -94,28 +94,45 @@ func (s *mockStsProviderAPI) AssumeRole(ctx context.Context, params *sts.AssumeR
 	return &aro, nil
 }
 
-var stsProviderApi aws.StsProviderAPI = &mockStsProviderAPI{}
-var ssmClient = &MockSsmClient{}
-
-func testCommand(args []string) error {
+func testCommand(args []string, cluster string) error {
+	if cluster != "" {
+		args = append(args, "--cluster", strings.TrimPrefix(cluster, "http://"))
+	}
```

Comment:
Does this mean that we are testing against a live fabric now? Should we tag this test as an integration test then?

Reply:
no it runs a test server from Go's `httptest` package which is running on localhost.